rasa_nlu/data_router.py

```diff
@@ -23,6 +26,20 @@
 logger = logging.getLogger(__name__)
 
 
+def deferred_from_future(future):
```

Comment:
Good find on the bridge-code for futures!

If there is ever an issue with concurrent ProcessPoolExector method, a (messier) Twisted direct way is covered here: https://stackoverflow.com/a/7215375/3334178